web/src/engine/keyboard/src/keyboards/keyboard.ts

```diff
@@ -454,23 +453,10 @@ export default class Keyboard {
     let _this = this;
 
     formFactors.forEach(function(form) {
-      // Currently doesn't work if we reset it to POLYFILLED, likely due to how 'calibration'
-      // currently works.
       _this.layoutStates[form] = LayoutState.NOT_LOADED;
     });
   }
 
-  public markLayoutCalibrated(formFactor: DeviceSpec.FormFactor) {
-    if(this.layoutStates[formFactor] != LayoutState.NOT_LOADED) {
-      this.layoutStates[formFactor] = LayoutState.CALIBRATED;
-    }
-  }
-
-  public getLayoutState(formFactor: DeviceSpec.FormFactor) {
-    return this.layoutStates[formFactor];
-  }
```

Comment:
Any use keeping this?

Reply:
Nothing is referencing it, and I don't see a major need to have it surfaced at this point.